### ISSUE TITLE
How to save the learner(contrastivelearner) as pytorch?

### ISSUE THREAD

**Issue by vimal-quilt** [koaning/embetter#88]
Thanks for the wonderful library. i want to save the learner as a pytorch module or to onnx graph. will u let me know how to do this?

**Comment by koaning** [comment]
I'd like to have a method for this at some point, but I'm not 100% sure what the cleanest solution would be for the entire API. 

Until then, you can [confirm in the code](https://github.com/koaning/embetter/blob/main/embetter/finetune/_constrastive_learn.py#L97) that a trained learner would have a `.network_` property that's a PyTorch object. You should be able to save that as you normally would.

**Comment by vimal-quilt** [comment]
Thanks for the reply. one more question. i see there is a accuracy drop in using this method of contrastive learner + logistic regression vs setfit method. will u comment something on this?

**Comment by koaning** [comment]
If you share the benchmark code I might be better able to comment. What happens if you compare with the `SbertLearner`? Theoretically that should be the same. Unless I made an implementation mistake or if I misunderstood their implementation.
